app/Http/Controllers/GroupController.php

```diff
@@ -63,6 +64,18 @@ public function postGroup($id = null)
             $group->addMember(Auth::user()->id, Group::ROLE_OWNER);
 
             if ($group->status === Group::STATUS_PENDING) {
+
+                // Send email to each admin to notify of new group needing approval
+                $admins = User::findByRoleName(Role::ROLE_ADMIN);
+
+                foreach($admins->all() as $admin) {
+                    Mail::queue('email.notification.verify_request_group', ['group' => $group], function ($message) use ($admin) {
+                        $message->subject('New Group Requesting Verification');
+                        $message->from('sayhello@opengovfoundation.org', 'Madison');
+                        $message->to($admin->email);
+                    });
+                }
+
```

Comment:
I'm not sure if I'm reading the code right, but it _looks_ like this will end up being called when the group is edited too, not just created?  If so, moving this up to line 53 will resolve the issue.